lib/typescript/ReactNativeZoomableView.d.ts

```diff
@@ -197,28 +197,15 @@ declare class ReactNativeZoomableView extends Component<ReactNativeZoomableViewP
      */
     _getNextZoomStep(): number | undefined;
     /**
-     * Zooms to a specific location in our view
+     * Zooms to a specific level and/or location in our view
      *
```

Comment:
Should specify what the x and y are, otherwise user has to find out experimentally or by reading internal code